docs/enterprise/getstarted/quickstart.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Quickstart
+description: Quickstart guides for lakeFS Enterprise
+parent: Get Started
+grand_parent: lakeFS Enterprise
+nav_order: 201
+---
+
+# Quickstart
+
+Follow these quickstarts to try out lakeFS Enterprise.
+
+{: .note }
+> ⚠️ lakeFS Enterprise Quickstarts are not suitable for production use-cases.
+{% include toc.html %}
+
+## lakeFS Enterprise Sample
```

Comment:
I find this section to be confusing, the next 2 are very clear - one is K8S and the other Docker-Compose.

What is "lakeFS Enterprise Sample"?  in What cases should I chose to use this? What is this based on if its not docker-cokpose and not k8s? 

So Im missing at the title level and generally the context on when I should following this path instead of say docer-compose?